src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconPoolApi.ts

```diff
@@ -2,18 +2,35 @@ import {IBeaconPoolApi} from "@chainsafe/lodestar-validator/lib/api/interface/be
 import {HttpClient} from "../../../api";
 import {IBeaconConfig} from "@chainsafe/lodestar-config";
 import {Attestation} from "@chainsafe/lodestar-types";
+import store from "../../../../ducks/store";
+import {signedNewAttestation} from "../../../../ducks/validator/actions";
+import {Json} from "@chainsafe/ssz";
 
 export class CgEth2BeaconPoolApi implements IBeaconPoolApi {
     private readonly httpClient: HttpClient;
     private readonly config: IBeaconConfig;
-    public constructor(config: IBeaconConfig, httpClient: HttpClient) {
+    private readonly publicKey?: string;
+    public constructor(config: IBeaconConfig, httpClient: HttpClient, publicKey?: string) {
         this.config = config;
         this.httpClient = httpClient;
+        this.publicKey = publicKey;
     }
 
     public submitAttestation = async (attestation: Attestation): Promise<void> => {
-        await this.httpClient.post("/eth/v1/beacon/pool/attestations", [
-            this.config.types.Attestation.toJson(attestation, {case: "snake"}),
-        ]);
+        const data = this.config.types.Attestation.toJson(attestation, {case: "snake"}) as Json & {
+            // eslint-disable-next-line camelcase
+            data: {beacon_block_root: string; index: string; slot: string};
+        };
```

Comment:
maybe it would be cleaner if you mannually convert to hex string from original attestation object. It would be more future proof in case json serialization changes